src/VirtoCommerce.Platform.Web/wwwroot/js/app/security/services/auth.js

```diff
@@ -144,14 +146,37 @@ angular.module('platformWebApp')
         authContext.userLogin = user.userName;
         authContext.fullName = user.userLogin;
         authContext.isAuthenticated = user.userName != null;
+        authContext.memberId = user.memberId;
+        authContext.iconUrl = null;
 
-        //Interpolate permissions to replace some template to real value
+        // Interpolate permissions to replace some template to real value
         if (authContext.permissions) {
             authContext.permissions = _.map(authContext.permissions, function (x) {
                 return $interpolate(x)(authContext);
             });
         }
+
+        if (authContext.memberId) {
+            getMemberInfo();
+        }
+
         $rootScope.$broadcast('loginStatusChanged', authContext);
     }
+
+    function getMemberInfo () {
+        modulesApi.query().$promise.then(function (modules) {
+                if (_.any(modules, module => module.id === 'VirtoCommerce.Customer')) {
+                    $http.get('api/members/' + authContext.memberId).then(function (memberResponse) {
+                        authContext.iconUrl = memberResponse.data.iconUrl;
+                    });
+                }
```

Comment:
Patform shoudn't know about specific module and shouldn't call its API directly.